prestodb/hdp2.6-hive/files/root/setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -ex
+
+# 0 make file system hostname resolvable
+echo "127.0.0.1 hadoop-master" >> /etc/hosts
```

Comment:
Hmm, isn't docker/docker-compose responsible for setting up network _and_ name resolution?

Note also that this is inherited by child containers.

Reply:
no, /etc/hosts are reset for every layer